external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaRelationSuite.scala

```diff
@@ -51,8 +51,8 @@ class KafkaRelationSuite extends QueryTest with SharedSQLContext with KafkaTest
     if (testUtils != null) {
       testUtils.teardown()
       testUtils = null
-      super.afterAll()
     }
+    super.afterAll()
```

Comment:
@kiszk Just wanted to understand. Is there a reason that its not under finally block ?

Reply:
Good catch, added `try-finally`